src/CommunityToolkit.Maui/CommunityToolkit.Maui.csproj

```diff
@@ -75,6 +75,7 @@
 
   <ItemGroup Condition="$(TargetFramework.Contains('-windows'))">
     <PackageReference Include="CommunityToolkit.WinUI.Notifications" Version="7.1.2" />
+    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.0.0" />
```

Comment:
do we need it since we already have it in Core lib?

Reply:
Don't bother about it, as I mentioned we will not merge this PR, it's just to show the api and how to use it